### ISSUE TITLE
How people have tried to solve Hospital acquired infection

### ISSUE THREAD

**Issue by genterist** [genterist/whiteWolf#4]

**Comment by mfarve1** [comment]
[http://www.ssmhealth.com/news/biovigil_ssm_st_mary/](url)
Really good article on how a hospital tracks when a nurse washes their hands. the technology explanation is below.  

**> But now, patients at SSM St. Mary’s Health Center know instantly if their nurse has clean hands due to the new device.  Upon entering the room the nurse’s badge changes colors and then chirps to remind the nurse that hand hygiene is required.  Once hands are sanitized, the badge illuminates a bright green hand symbol which signals to all that the hands are clean.  During the past two months, staff hand-washing compliance on two test units at the hospital are now routinely maintained near 100 percent (99% and 97%), a level never before attained by a hospital, especially with this type of visibility and precision.  While it’s still too early to determine if the findings are statistically significant, the initial results are very exciting and the hospital plans to continue with the testing to collect more data and conduct final research statistics.
> 
> The collaborative test phase for the system, developed by Biovigil Hygiene Technologies in Ann Arbor, Mich., began last April at SSM St. Mary’s Health Center in St. Louis as their first test site. A tiny computer and built-in sensor technology in the badge detects chemical vapors from alcohol-based hand washing solutions used throughout the hospital (think of it as a tiny smoke detector designed to detect clean hands).  When going in or out of a patient’s room, the badge cycles through a series of color changes from green to flashing yellow to flashing red. Green means clean.  To return the badge to the green (clean) status, a staff member simply places their sanitized hands close to the badge as they are drying – proudly called “pledging the badge.”  Each “pledge” is recorded electronically via a web-based information system.**

**Comment by mfarve1** [comment]
![image](https://cloud.githubusercontent.com/assets/25184593/22121276/6b30034e-de51-11e6-9b10-62bef531a05c.png)

**Comment by mfarve1** [comment]
This is similar to the above article, Hospital tracks how nurses wash their hands
[ProQuestDocuments-2017-01-19.pdf](https://github.com/genterist/whiteWolf/files/717534/ProQuestDocuments-2017-01-19.pdf)

**Comment by mfarve1** [comment]
Some statistics From - Economics and Preventing Hospital-acquired Infection
Nicholas Gravescorresponding author*

Approximately 1 in 10 hospitalized patients will acquire an infection after admission, resulting in substantial economic cost (1). The primary cost is that patients with hospital-acquired infections have their stay prolonged, during which time they occupy scarce bed-days and require additional diagnostic and therapeutic interventions (2). Estimates of the cost of these infections, in 2002 prices, suggest that the annual economic burden is $6.7 billion per year in the United States (3)1 and £1.06 billion (approximately $1.7 billion) in the United Kingdom (4).

**Comment by mfarve1** [comment]
[Feeding Back Surveillance Data to Prevent HAI.pdf](https://github.com/genterist/whiteWolf/files/717609/Feeding.Back.Surveillance.Data.to.Prevent.HAI.pdf)

Another paper that talks about using surveillance of data to reduce hospital acquired infections.  This is somewhat similar to my idea that I had about creating a threat matrix

**Comment by genterist** [comment]
From:
Wachter, Robert M. Understanding patient safety. New York: McGraw-Hill Medical, 2008.

WHAT PEOPLE HAVE TRIED (using software):
+ General goals are:
   . Improving communication
   . Making knowledge more readily accessible
   . Prompting for key pieces of info
   . Assisting with calculations
   . Monitoring and checking in real time
   . Providing decision support
+ Electronic medical record
  . Issues with data entry (the cut-n-paste culture, the speed, the cost if hospitals want to "wire-in" hardwares)
+ Computerized provider order entry (CPOE)
   . Reduce medical errors by 55% if combined with a decision support system
   . Can automatically raise red-flags
   . More advantages are in table 13-2, pg 140
   . Issues with time delays/gaps in medication delivery (system down time, outdated database of drugs)
+ Bar coding and Radio frequency identification systems
   . Prevent data entry errors
   . Issues with costs (RFID), multiple drugs from different patients in one cart
+ Smart intravenous pumps
+ Computerized decision support
   . Provides alerts on drug-drug, drug-allergy, drug-lab interactions
   . Provide alerts when patient vital signs go out of certain parameters
   . Automatically enforce standards on medical electric devices (such as electric pumps)
+ Clinical artificial intelligence
   . Support diagnostic decisions
   . Predicting future problems

THINGS TO WATCH OUT (when implement software)
+ Alert fatigue
+ "Cookbook medicine" culture
+ Low confidence and adoption rate of new techs

**Comment by genterist** [comment]
Team,

Based on the recent results of our surveys, I did a little bit more research on the issue and here is what I found.

+ Human factor engineering
According to our survey, one of the main reasons why HAI occurred is because the nurses have many things to do or trainers were not completely aware of all potential issues, mistakes that are overlooked. This video gives an example of how easy it is to overlook the human factor https://www.youtube.com/watch?v=O9aD-bBx8TA

So I think about ways to improve their work flows and it turns out that it is an extremely complicated problem. This evidence table shows the past efforts and various results when people try to improve quality of care by changing work flows (in most cases, by employing new information technologies) https://www.ncbi.nlm.nih.gov/books/NBK2638/table/ch31.t1/?report=objectonly

+ Time motion study
Looks like lots of improvements can be made to improve work flows if time-motion studies are done and done correctly. For example, is the position of med vault optimal for nurses? Why a nurse took a patient in and out of a X-ray room 2 times ? With time motion study, we can look at what happened before a HAI occurred. I will have to look more into that. 

Extra links:
https://www.ncbi.nlm.nih.gov/books/NBK2638/
https://www.youtube.com/watch?v=8QgK07J3gio
